### ISSUE TITLE
[NGRM] - In SyncLogViewer initial mount sometimes has wrong zoom when sync scale by controllers

### ISSUE THREAD

**Issue by mirisb** [equinor/webviz-subsurface-components#2108]
While mounting the component we fetch data from server which takes some. while the data is being fetched we first pass an empty log in the props:
```
const emptyLog : WellLog = {
      header: { startIndex: 0, endIndex: 1, step: 1 },
      curves: [
        {
          name: "MD",
          unit: "m",
          dimensions: 1,
        },
        {
          name: "Empty",
        },
      ],
      data: [
        [0, null],
        [1, null],
      ],
    };
```
Once fetching from server is complete the props are replaced with the real props that contain real logs from several wells.

We sync scales between different controllers in SLV using onContentRescale callback from the component (usually this function is only called when use flattening and want to apply our own logic for track positioning):
```
   const onContentRescale = React.useCallback(
     (iView: number) => {
       if (controllers.length <= iView) {
         return;
       }

       // we want to mantain the same aspect ratio - thus we take the scale from the affected well and apply it to all the wells
       const scale = controllers[iView].getContentScale();
       if (!isFinite(scale)) {
         return;
       }

       controllers.forEach(controller => {
         if (controller == controllers[iView]) {
           return; // skip controller which caused the onContentRescale
         }
        controller.setContentScale(scale);
       });
    },
    [controllers]
  );
```
The problem occurs when sometimes when calling controller.setContentScale we get the correct scale however the zoom itself is calculated according to the size of the overlay, in function getContentBaseScale () in WellLogView.ts (in Webviz codebase). When this function is called, the overlay has not yet become visible, its size not yet calculated and it contains a bogus height which causes it to give wrong results which leads to calculation of wrong zoom.

![image](https://github.com/equinor/webviz-subsurface-components/assets/18436935/aa5aa32b-d8b1-414d-8daa-0d96646661ec)

This is easily recreated in Viz storybook by mounting story "Two Wells Same Log" and after it is mounted correctly switch to story "Two Wells Different Log". Since this bug depends on timing and fetching data from server, to recreate the storybook must be reloaded from scratch to clear caches.

**Comment by mirisb** [comment]
@Vladimir-Kokin , maybe a solution would be to add a query to the controller to make sure that it is fully rendered.

In debugger I was able to display the overlay and see that it is not visible:
![image](https://github.com/equinor/webviz-subsurface-components/assets/18436935/bafae4ce-e760-455f-8b05-41a152eee15f)

If the controller has a query function to make sure it is rendered, we could test it before trying to change the scale...

**Comment by mirisb** [comment]
This seems to have been fixed by #2114
